This patch is only applicable to special sound driver of this project.

---
diff --git a/audio/audio_driver.c b/audio/audio_driver.c
index f1dd4e0..ed7c763 100644
--- a/audio/audio_driver.c
+++ b/audio/audio_driver.c
@@ -1886,6 +1886,11 @@ void audio_driver_menu_sample(void)
    if ((audio_st->flags & AUDIO_FLAG_SUSPENDED))
       check_flush                         = false;
 
+   for (int i = 0; i < 1024; i++)
+   {
+      samples_buf[i] = -32768;
+   }
+
    while (sample_count > 1024)
    {
       if (  recording_st->data   &&
